toolkit/tools/imagegen/configuration/configuration.go

```diff
@@ -93,6 +95,33 @@ func (c *Config) GetDiskContainingPartition(partition *Partition) (disk *Disk) {
 	return nil
 }
 
+// GetKernelCmdLineValue returns the output of a specific option setting in /proc/cmdline
+func GetKernelCmdLineValue(option string) (cmdlineValue string, err error) {
+	const cmdlineFile = "/proc/cmdline"
+
+	content, err := os.ReadFile(cmdlineFile)
+	if err != nil {
+		logger.Log.Errorf("failed to read from %s", cmdlineFile)
+		return
+	}
+
+	cmdline := string(content)
+	if strings.Count(cmdline, option) > 1 {
+		err = fmt.Errorf("/proc/cmdline contains duplicate %s entries, which is invalid", option)
```

Comment:
>fmt.Errorf("/proc/cmdline contains duplicate %s entries, which is invalid", option) [](http://example.com/codeflow?start=8&length=83)

General hint: always good to surround strings with quotes or brackets in messages. Every now and then an empty string may end up being passed and the message with lose clarity. Empty quotes or brackets at least indicate where you'd expect something more to appear. Usually speeds up debugging.

Reply:
Fixed